app/services/catalog/order_item_transition.rb

```diff
@@ -0,0 +1,50 @@
+module Catalog
+  class OrderItemTransition
+    attr_reader :order_item_id
+    attr_reader :state
+
+    def initialize(order_item_id)
+      @order_item_id = order_item_id
```

Comment:
@lindgrenj6 Should we just fetch the order_item object here and keep that around

```
@order_item = OrderItem.find(order_item_id)
```